collections/made-in-india/index.md

```diff
@@ -1,16 +1,19 @@
 ---
 items:
+ - TheAlgorithms/Python
  - hoppscotch/hoppscotch
+ - anuraghazra/github-readme-stats
  - hasura/graphql-engine
  - geekyants/nativebase
- - TheAlgorithms
```

Comment:
Does this not display properly as an organisation?

Reply:
It displays but sometimes it or other things disappear. See [this comment](https://github.com/github/explore/pull/2197#issuecomment-810565964) It was unanswered for a specific reason so I replaced this org with 2 of its really popular repos

Reply:
Shows up fine for me; let's leave it as-is, thanks!

Reply:
Done @MikeMcQuaid